cdap-tms/src/test/java/co/cask/cdap/messaging/store/DataCleanupTest.java

```diff
@@ -170,7 +170,7 @@ public void testOldGenCleanup() throws Exception {
 
       metadataTable.deleteTopic(topicId);
       // Sleep so that the metadata cache in coprocessor expires
-      TimeUnit.SECONDS.sleep(1);
+      TimeUnit.SECONDS.sleep(3);
```

Comment:
It would be good to add the JIRA in the comment for future reference, since this is an unrelated change for this PR.